core/services/workflows/v2/config.go

```diff
@@ -57,8 +59,10 @@ type EngineLimits struct {
 	MaxTriggerSubscriptions             uint16
 	TriggerEventQueueSize               uint16
 
-	MaxConcurrentWorkflowExecutions uint16
-	MaxConcurrentCapabilityCalls    uint16
+	MaxConcurrentWorkflowExecutions         uint16
+	MaxConcurrentCapabilityCallsPerWorkflow uint16
+	WorkflowExecutionTimeoutMs              uint32
+	CapabilityCallTimeoutMs                 uint32
```

Comment:
Why not use `time.Duration` for these? Sheds the conversion on use, and makes the literals more readable.

Reply:
this pattern is used throughout the `capabilities` and `gateway` packages, can move to a separate discussion because changing here makes this now inconsistent with other areas of the code.